docs/conf.py

```diff
@@ -14,7 +14,7 @@
 # Keep capitalization due to similar linking on GitHub's markdown preview.
 shutil.copy2('../CHANGELOG.md','./CHANGELOG.md')
 
-latex_engine = "xel  atex"
+latex_engine = "xelatex"
```

Comment:
@samjwu We should add this word to the wordlist

Reply:
this is in conf.py, which is not checked

Reply:
Sorry, this comment wasn't for this PR. Will delete. 

Thanks for clarifying, Sam.